date/date_c.php

```diff
@@ -313,15 +313,15 @@ public function __wakeup() { }
 class DateTime implements DateTimeInterface {
     /**
      * (PHP 5 &gt;=5.2.0)<br/>
-     * @link https://www.php.net/manual/en/datetime.construct.php
+     * @link https://secure.php.net/manual/en/datetime.construct.php
```

Comment:
https://www.php.net/manual/en/datetime.construct.php works
According to https://github.com/JetBrains/phpstorm-stubs/pull/502#issuecomment-461512744 It's preferable not to hardcode https://secure.php.net/ instead of https://www.php.net/

Reply:
Updated url to not use secure subdomain, but still removed the www subdomain. httpS with www subdomain does NOT work.